python/datasources/cdc_vaccination_national.py

```diff
@@ -45,24 +56,26 @@
 # and they don't publish these population numbers directly anywhere, so I am
 # taking the population percentages directly off of the chart here:
 # https://covid.cdc.gov/covid-data-tracker/#vaccination-demographic
-CDC_AGE_GROUPS_TO_POP_PCT = {
-    'Ages_<2yrs': '2.3',
-    'Ages_2-4_yrs': '3.6',
-    'Ages_5-11_yrs': '8.7',
-    'Ages_12-17_yrs': '7.6',
-    'Ages_18-24_yrs': '9.2',
-    'Ages_25-49_yrs': '32.9',
-    'Ages_50-64_yrs': '19.2',
-    'Ages_65+_yrs': '16.5',
-    'US': '100',
+AGE_GROUPS_TO_POP_PCT = {
```

Comment:
Should we be periodically checking/updating these numbers manually?

Reply:
Yeah probably, definitely an imperfect soltion